pkg/resources/user_test.go

```diff
@@ -8,7 +8,6 @@ import (
 	sqlmock "github.com/DATA-DOG/go-sqlmock"
```

Comment:
this file is a remnant of some of the older tests which we are moving away from. this entire file can be deleted

Reply:
ok